packages/shylu/shylu_dd/frosch/test/CMakeLists.txt

```diff
@@ -7,5 +7,9 @@ ADD_SUBDIRECTORIES(
     LocalPartitionOfUnityBasis
 #    InterfacePartitionOfUnity
     TwoLevelPreconditionerInputFiles
-    TwoLevelPreconditionerBlockInputFiles    
+    Thyra_Xpetra
+    Thyra_Xpetra_Prec
+    Thyra_Tpetra
```

Comment:
What are the differences between `Thyra_Xpetra` and `Thyra_Xpetra_Prec`?

Reply:
Thyra_Xpetra_Prec is a test for using the interface in LifeV.  
I will disable the test for it is similar to Thyra_Xpetra.